ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v2/aemform/aemform.html

```diff
@@ -45,10 +45,13 @@
 <sly data-sly-use.clientLib="${'/libs/granite/sightly/templates/clientlib.html'}"/>
 <sly data-sly-test.useIframe="${form.useIframe != 'false'}">
     <sly data-sly-test.formLocaleString="${form.locale || 'en'}"/>
-        <iframe class="cmp-aemform__iframecontent"
-            src="${request.contextPath}${resource.path}.iframe.${formLocaleString}.html?dataRef=${request.requestParameterMap['dataRef'] != null ? request.requestParameterMap['dataRef'][0].toString : '' @ context='attribute'}${!wcmmode.edit ? '&wcmmode={0}':'' @ format=[wcmmode.toString]}"
-            width="100%;"
-            data-form-page-path="${form.formEditPagePath}"></iframe>
+    <sly data-sly-set.dataRef="${request.requestParameterMap['dataRef'] ? request.requestParameterMap['dataRef'][0].toString :'' @ context='attribute'}"/>
+    <sly data-sly-set.wcmmode="${!wcmmode.edit?'wcmmode=DISABLED':''}"/>
+    <sly data-sly-set.formUrlParams="?${wcmmode}${dataRef ? '&dataRef=' :''}${dataRef}"/>
```

Comment:
formUrlParams would have `?` if wcmmode is edit and dataRef is empty, can you fix this, you can check my previous comment

Reply:
@sakshi-arora1 told that it is in iframe therefore it will never open in edit mode

Reply:
Changes have to be self contained, let's not assume any specific use-case's.

Reply:
> formUrlParams would have `?` if wcmmode is edit and dataRef is empty, can you fix this, you can check my previous comment

'?' is there in the starting of the formUrlParams then wcmmode is included and then & is appended based on whether the value is present or not